src/locales/en.json

```diff
@@ -21,6 +24,10 @@
     }
   },
   "button": {
+    "addPanel" : {
+      "text" : "Add Panel",
+      "tooltip" : "Add New Panel"
```

Comment:
"Add to Raw Panel Files" in the tooltip instead.

Reply:
done